gns3fy/gns3fy.py

```diff
@@ -1897,3 +1898,25 @@ def restore_snapshot(self, name=None, snapshot_id=None):
 
         # Update the whole project
         self.get()
+
+    def arrange_nodes(self):
+        """
+        Re-arrgange the existing nodes
+        in a geometric fashion (circle)
+
+        **Usage**
+
+        Project.arrange_nodes()
```

Comment:
Like the initiative. The main part is referencing what the method **does** and the next part Usage can be replaced with Example and encompass it in Mardown python code.

Also it is needed to specify the attributes that are needed for the method to run. Look at the examples of other methods, better yet the `Gns3Connector`

https://github.com/davidban77/gns3fy/blob/73d1c48663419ef8d52b9b091db97829665af75e/gns3fy/gns3fy.py#L48